docs/demo.mdx

```diff
@@ -0,0 +1,506 @@
+
+:::tip
+
+If you see `zsh: command not found` (or a similar message), please go back to
+[Setting up the environment](../getting-started.mdx#setting-up-the-environment) section
+where we were redirecting you to the official **React Native CLI Quickstart**, and make sure you
+have completed it.
+
+Provided you have `ANDROID_SDK_ROOT` (or `ANDROID_HOME`) environment variables
+defined, using an explicit path to `emulator` should help:
+
+<Tabs groupId="desktopOS">
+  <TabItem value="darwin" label="MacOS">
+    <CodeBlock language="bash">$ANDROID_SDK_ROOT/emulator/emulator -list-avds</CodeBlock>
+  </TabItem>
+  <TabItem value="linux" label="Linux">
+    <CodeBlock language="bash">$ANDROID_SDK_ROOT/emulator/emulator -list-avds</CodeBlock>
+  </TabItem>
+  <TabItem value="win32" label="Windows">
+    <CodeBlock language="plain text">%ANDROID_HOME%\emulator\emulator -list-avds</CodeBlock>
+  </TabItem>
+</Tabs>
+
+:::
+
+#### _BELOW is an admonition with plain text examples, code spans and emoji:_
+
+:::info
+
+If you wish to use an attached Android device (via USB or wireless network), make sure it is available:
+
+```bash
+adb devices
+```
+
+If the device is properly connected, you should get an output like this:
+
+```plain text
+List of devices attached
+2F6315NVPH      device
+```
+
+Assuming the device is available (troubleshooting _adb_ issues is out of scope of this tutorial), you just
+need to use `android.att.*` Detox configurations instead of `android.emu.*` where required.
+
+:warning: If you have multiple devices connected, replace that loose regular expression (`adbName: '.*'`)
+in your Detox config with a specific device name.
+
+:::
+
+#### _BELOW is an admonition with JS code blocke:_
+
+:::tip
+
+It is a good idea to start every test from a fresh state, since the preceeding ones might leave your application
+in an unpredictable state if they fail.
+
+One way to do it is to launch the app as a new instance in `beforeEach` hook instead:
+
+```js
+beforeEach(async () => {
+  await device.launchApp({ newInstance: true });
+});
+```
+
+The other way is to _reload React Native_ without restarting the app. Like any live reloading, it is apt to cause glitches for more complex apps,
+but for simpler apps it proves to be a quicker way to reset the state between the tests:
+
+```js
+beforeEach(async () => {
+  await device.reloadReactNative();
+});
+```
+
+So, pick your favorite one wisely, on the basis of _speed_ vs _stability_ considerations.
+
+:::
+
+## Tabs with code spans:
+
+<Tabs groupId="configurationName">
+  <TabItem value="ios.sim.debug" label="npm">
+    <CodeBlock language="sh">npm i --package.js</CodeBlock>
+  </TabItem>
+  <TabItem value="ios.sim.release" label="yarn">
+    <CodeBlock language="sh">yarn add axios </CodeBlock>
+  </TabItem>
+</Tabs>
+
+## CodeBlocks, line spacings, links, images, tables, code fragments, headings section, lists:
```

Comment:
Do we have a table here? I couldn't find it

Reply:
Please don't use long headlines, let's use one headline for each component we're presenting

Reply:
It will be easier to navigate in the demo doc from the right menu

Reply:
Table added!
Headlines shortened for each component only.
The TOC on RHS now looks much better.